Library/Homebrew/Gemfile.lock

```diff
@@ -182,7 +182,6 @@ GEM
     zeitwerk (2.6.12)
 
 PLATFORMS
-  aarch64-linux
```

Comment:
Will add a workaround in `brew vendor-gems` for this as I found a one-liner that will undo what Dependabot is doing here.